fb_reels_publishing_api_sample/index.js

```diff
@@ -273,14 +279,26 @@ app.get("/listUploadedVideos", async function(req, res) {
                             if (value['updated_time']) {
                                 self[index]['updated_time'] = (new Date(value['updated_time'])).toLocaleString();
                             }
+                            if (isStories && value['creation_time']) {
+                                self[index]['creation_time'] = (new Date(value['creation_time'] * 1000)).toLocaleString();
+                            }
                         })
                         // Render the Upload page, but now send back the list of past reels of the selected page
```

Comment:
is there supposed to be a difference between these two branches under if (isStories) ?

Reply:
reels has a field called `updated_time` which is in milliseconds while stories has a field called `creation_time` which is in seconds. Thus you need to multiply it by 1000